recipes/gyptis/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "gyptis" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "f0a2ac4de0e18ee73de6930380caff3e35651faa1cc79baabebddedc910ea33d" %}
```

Comment:
You don't need a jinja variable for something that us used only once.

Reply:
Okay I will remove the variable